### PR TITLE
Updates to XML 7, package versions, and container tags

### PR DESCRIPTION
**Pull request by srp3rd**

### CODE REVIEW

File: bin/run_atarchiver.sh [hunk 1]
```diff
@@ -4,28 +4,42 @@ source $loc/base_init.sh
...
 fi
 
 lsst_dds_partition_prefix=$1
 container_version=$2
+network_interface_address=$3
+domain_id="0"
+if [ -n $4 ];
+then
+    domain_id=$4
+fi
```
Comment (mxk62) [inline bin/run_atarchiver.sh:24]: Assigning a default value to a variable if the explicit value is not provided could also be done with
```bash
domain_id=${4:-0}
```
However, at the end of the day it's just a matter of personal preferences, so I'm leaving the decision which one to use up to you.

File: bin/run_atcontroller.sh [hunk 2]
```diff
@@ -4,25 +4,16 @@ source $loc/base_init.sh
...
 fi
 
-if [ -z $2 ];
-then
-    echo "$0: missing argument: lsst_dds_partition_prefix container_version"
-    exit 1
-fi
-
-lsst_dds_partition_prefix=$1
 container_version=$2
```
Comment (mxk62) [inline bin/run_atcontroller.sh:11]: To my understanding, after the changes, script's first and only positional argument is the container version, right? If so, shouldn't the line 11 read
```bash
container_version=$1
```

**Review by mxk62 — APPROVED** [review]

File: bin/run_atarchiver.sh (hunk 1 above)
Comment (srp3rd) [reply bin/run_atarchiver.sh:24]: I usually do it that way to make it a bit more clear what's being done.

File: bin/run_atcontroller.sh (hunk 2 above)
Comment (srp3rd) [reply bin/run_atcontroller.sh:11]: Yes.  I'll address this.

**Pull request merged**